File: auto-docs/executables/r/stacked_bar.r
# Learn about API authentication here: plot.ly/r/getting-started
# Find your api_key here: plot.ly/settings/api

library(plotly)

py <- plotly(username='TestBot', key='r1neazxo9w')
trace1 <- list(
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(20, 14, 23), 
  name = "SF Zoo", 
  type = "bar"
)
trace2 <- list(
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(12, 18, 29), 
  name = "LA Zoo", 
  type = "bar"
)
data <- list(trace1, trace2)
layout <- list(barmode = "stack")
response <- py$plotly(data, kwargs=list(layout=layout, filename="stacked-bar", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/nodejs/stacked_bar.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')
var trace1 = {
  x: ["giraffes", "orangutans", "monkeys"], 
  y: [20, 14, 23], 
  name: "SF Zoo", 
  type: "bar"
};
var trace2 = {
  x: ["giraffes", "orangutans", "monkeys"], 
  y: [12, 18, 29], 
  name: "LA Zoo", 
  type: "bar"
};
var data = [trace1, trace2];
var layout = {barmode: "stack"};
var graph_options = {layout: layout, auto_open: false, fileopt: "overwrite", filename: "stacked-bar"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});